oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/geometry/OSHDBGeometryBuilder.java

```diff
@@ -279,24 +279,31 @@ private static Geometry getMultiPolygonGeometry(
       T entity, OSHDBTimestamp timestamp, TagInterpreter areaDecider, P clipPoly) {
     Geometry geom = OSHDBGeometryBuilder.getGeometry(entity, timestamp, areaDecider);
     return Geo.clip(geom, clipPoly);
+  } 
+  
+  /**
+   * Create the Geometry of an OSHDBBoundingBox. Will always contain 5 Coordinates, even for
+   * point-like bbx -> point-like polygons for bbx of a Point.
+   *
+   * @param bbox The BoundingBox the polygon should be created for.
+   * @return com.vividsolutions.jts.geom.Geometry Returns JTS geometry object for convenience.
+   * Result is an empty polygon for a null BBX.
```

Comment:
i think this should be indented according to our style guide. also, I think we never use the abbreviation _BBX_ for bounding boxes, but rather use _bbox_ if we want to have something short…

Reply:
oh, and is there a reason to mention the class of the result object here in the `@return` javadoc? I mean, doesn't javadoc see that from the definition of the function itself already?

Reply:
just copied the javadoc that was in the header before. Was wondering, too :-)